db/migrations/20230307112034_create_orders_table.sql

```diff
@@ -0,0 +1,17 @@
+-- migrate:up
+CREATE TABLE orders (
+ id INT NOT NULL AUTO_INCREMENT PRIMARY KEY,
+ user_id INT NOT NULL,
+ order_id INT NOT NULL,
```

Comment:
order_number로 컬럼명 수정해 주시고 varchar로 바꿔주세요. 크기는 100정도면 좋을 것 같습니다.
```suggestion
 order_number VARCHAR(100) NOT NULL,
```